transformer_engine/pytorch/utils.py

```diff
@@ -12,6 +12,11 @@
 import transformer_engine.pytorch.cpp_extensions as ext
 
 
+def maybe_contiguous(tensor: torch.Tensor) -> torch.Tensor:
+    """Make tensor contiguous if final stride is not 1."""
+    return tensor.contiguous() if tensor.stride(-1) != 1 else tensor
```

Comment:
Is `maybe_contiguous` used outside of `attention.py`? It seems weird and non-general, so I'd prefer not exposing it in the utilities if possible.